client/blocks/design-sidebar/index.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import RootChild from 'components/root-child';
+
+class DesignSidebar extends Component {
+	constructor( props ) {
+		super( props );
+	}
```

Comment:
Hm. Good question! Apparently it is, which I'd expect, but I'm never sure about javascript "classes".